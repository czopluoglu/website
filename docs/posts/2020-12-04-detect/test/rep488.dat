11000010000000001000
10101101011011011110
01110011111111111111
10101101011011101111
01100001110011000111
10110001001010010011
11001010101010100110
00000000111100101111
11000100011101111110
11000000000000101100
01010101101010111011
10101111111010010111
11011110111001010110
10000001000000111111
11001101111000100011
11001011111101111111
00001000001110110100
10000100110001100000
11010100011111000110
00000001111100001000
10111100111011100111
01010111011000000110
00001001001010000110
10011101011101110111
10111100001110010010
10000001011000010011
01011100100011100111
10001001110110100111
11110111111111101111
11000010010000011111
00101101000010011100
00011101000011010110
00100010101010111001
11001101111010010001
10101000110001101111
11001111001000010010
11010001101110100110
00010101001111110110
10110001111010101110
10000001000101100011
10011110101011111101
11001101010001000110
11100100011000110110
10111011110101101111
01110000011011101111
11110110111011101111
00101100011111110011
10010100000010000011
00011010111101101111
00000001000010101001
00100111100110001111
10011011111010111111
10001100001000000110
10000001001111110010
01001101111011001011
10010101110010000000
00000101001111111111
11000011111111111111
10000111111000111101
00000100011100011111
00001001011110001000
10110010111000100000
11000001001111001110
11000001101011110011
00011101100000101010
00000000011010110101
10111110111011011110
11010111100000110000
11010000101000111001
00011001011110100000
10111011001111011111
11011101001010000101
10001101011100111110
01111101101110111111
10000000001011100110
01111011101100111111
10001001100010000110
10100111111111100100
00111011010110111010
10111011100110000011
01000101001001100111
00000001001010100010
11100110111011000111
01100110011000011010
10100101000111101110
11001001101111110110
00010001001011100010
11011101010001100011
01101101111110011111
11001101000010101111
01011001011000111101
01001000101100101011
11110001000010100111
11110100111000011010
10110100101011111100
10010111011111110001
00110001101001101011
11101100001101101001
01101000001010111011
11111111011001111110
01000001001001100101
00110101001010011111
10010010010111100000
00010011111111101110
01101101110010100110
10101100011110101111
00100001111011101111
11111011110010000100
11001101011011111100
11001111111111111111
00010001010010110010
10000001000000001010
00001011110111101111
10111111100010001010
00100000000110000110
00000000001000100111
10110101111010111010
10000000111011000010
10011110100000001111
00111111111010011111
01111011111111111111
01100011100010111111
11011001000011100100
00000011001101011000
00100100001111111111
01111111110011110101
10001111011101101110
10010110010001000100
10111110111101111111
11100011101100110000
10001001110110001111
00000010011111111110
01001010010110101111
01000010111111110110
10010101100110111111
01011000111011100110
01010100111111111111
00000001100000000000
11011011110111111111
10111011110000000001
01100000100000001000
00101100000000000010
01000111111110010111
11010111100000100110
00000101010110110011
00001001001110111111
01001101001000100111
01000100011000000010
01111100111101111111
00001001011010111100
00001000011110100010
00011100001000000011
10011101000011111001
11110010111001011110
10001011111010101111
10000100010011010111
11111011101101011110
00010000001100100001
01101000000100100110
00000000101100000000
11010101110011111111
11111111111011111111
01100100001111010010
00101101011001001100
11000100101011110111
00001001001101000010
10111011000100001101
00100101100111101111
11111111111001101111
00000010000000101001
00000000101101110011
11001011011011110011
01101100100110100111
11110101110010001111
00000101101000101011
00100001011011100011
11100011111000111111
00111100000111010111
11101001101111101101
11110000000010110100
00100100101011001011
11000101011110011110
10010110110010001010
01111101111001011011
00000001101101111011
10100011001001101011
01000001010010110101
10010101101010000101
10001111001010100010
00110110110000110011
11110010111110101010
01111110010100000110
11010101111111101110
01001000110010101010
10101001001101001001
00111101010100011100
10011110111010111111
00111101101010101100
10000111111010000111
11110011100000100010
00100111011000111011
01100001101011101111
01000111111010100110
11010010000101101100
01101001101110111110
10101101111011001111
10111101111010000111
00001001000000000011
01000000001101110011
10100000101001010011
01110001011100110011
00111101111111111111
11111001111010000111
10111111111011101111
10101000111111110110
11000011110100101110
10001000001010100110
10111111110110100011
11100110101100001010
11111101011110111111
10101001001011000100
10011000101110110100
10110001111111111110
00000000010000100110
00110110011111110111
01100001101110110111
11000101111110000011
00110111101111111110
00000101100000110110
00001100001110111111
10101001011101101010
10110000111000101111
00010110011111100001
00000100001110101010
10001100001010101111
00001001001110000101
10010111011010111011
10010000011110011000
00000010010011000010
11111100011110111111
11111011000010101100
00010000001101001011
00010110011011101111
01001001111101110111
11011101111011100110
11101000000111010010
10011110110101101110
11000000000001110110
10110100101011111010
00001011110110101111
