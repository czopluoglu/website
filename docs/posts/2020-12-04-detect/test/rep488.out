-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep488.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.40

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.11
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7596
                   IDN index value:   0.7000
                           Ratio r:   0.6467

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    1    1    3    1    1    3    1    1
    4    4    3    4    4    4    4    3    4    4

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    3    4    6    7    9   10

   -----------CLUSTER  2-------------

    2

   -----------CLUSTER  3-------------

    5    8   13   18

   -----------CLUSTER  4-------------

   11   12   14   15   16   17   19   20
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d-+++++++--+--+-----
   -d+--+--+-+------++-
   ++d++++---+---------
   +-+d--+-+-+++---+-+-
   +-+-d++----------+--
   +++-+d++-++----+-+++
   +-++++d+---++-+---+-
   +----++d---+--+++-+-
   ++-+----d+++-+---+--
   -----+--+d+------+-+
   -+++-+--++d+----+--+
   +--+--+++-+d--++-++-
   ---+--+-----d+++++++
   --------+---+d+++++-
   +-----++---+++d+--+-
   -----+-+---++++d++-+
   ---+---+--+-++-+d+-+
   -+--++--++-+++-++d-+
   -+-+-+++---++++---d+
   -----+---++-+--++++d
 
 No cross validation for this DETECT run
